src/index.ts

```diff
@@ -143,3 +147,53 @@ export function submitEmail(fn: (target, widget, email) => any): void {
 }
 
 if (window) asyncLoad();
```

Comment:
More isometric way of checking this.

```suggestion
if (typeof document !== "undefined") asyncLoad();
```